DeusEx/Classes/DXRFixup.uc

```diff
@@ -264,6 +267,19 @@ function NYC_04_AnyEntry()
     }
 }
 
+function UNATCO_05_AnyEntry()
+{
+    local DeusExMover d;
+    switch (dxr.localURL)
+    {
+        case "05_NYC_UNATCOHQ":
+            foreach AllActors(class'DeusExMover', d, 'supplydoor') {
+                class'DXRKeys'.static.MakeDestructible(d);//I need to move this into DXRKeys config
```

Comment:
or maybe this should be considered a fix for a possible annoyance in the original game too?